sematic/resolvers/cloud_resolver.py

```diff
@@ -38,7 +38,7 @@ class CloudResolver(LocalResolver):
 
     Parameters
     ----------
-    detach: Optional[bool]
+    detach: bool
```

Comment:
From [typing.Optional](https://docs.python.org/3/library/typing.html#typing.Optional):

> Optional[X] is equivalent to X | None (or Union[X, None]).
>
> Note that this is not the same concept as an optional argument, which is one that has > a default. An optional argument with a default does not require the Optional qualifier > on its type annotation just because it is optional.